demo_formats.py

```
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+"""
+Demo script to showcase markdown_lab's multiple output formats.
+"""
+
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)